tests/test_smoke.py

```diff
@@ -705,6 +705,24 @@ def test_spot_storage():
         run_one_test(test)
 
 
+# ---------- Testing spot TPU ----------
+def test_spot_tpu():
+    """Test managed spot on TPU."""
+    name = _get_cluster_name()
+    test = Test(
+        'test-spot-tpu',
+        [
+            f'sky spot launch -n {name} examples/tpu/tpuvm_mnist.yaml -y -d',
+            'sleep 5',
+            f's=$(sky spot status); printf "$s"; echo; echo; printf "$s" | grep {name} | head -n1 | grep STARTING',
+            'sleep 600',  # TPU takes a while to launch
+            f's=$(sky spot status); printf "$s"; echo; echo; printf "$s" | grep {name} | head -n1 | grep "RUNNING\|SUCCEEDED"',
+        ],
+        f'sky spot cancel -y -n {name}',
```

Comment:
We don't need a tear down command for managed spot jobs.

Reply:
Oh this smoke test will pass if the job changed from `STARTING` to `RUNNING`. In this case, we can actually kill the job to save some costs. But we can remove it if we want as the job is short.

Reply:
Oh I thought it was short - let's still keep it then.